test/solidity-ibc/utils/RefIBCERC20.sol

```diff
@@ -0,0 +1,93 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.28;
+
+import { ERC20Upgradeable } from "@openzeppelin-upgradeable/token/ERC20/ERC20Upgradeable.sol";
+import { IMintableAndBurnable } from "../../../contracts/interfaces/IMintableAndBurnable.sol";
+
+/// @title Reference IBC ERC20 Contract
+/// @notice This implementation is intended to serve as a base reference for developers creating their own
+/// IBC-compatible ERC20 tokens.
+contract RefIBCERC20 is ERC20Upgradeable, IMintableAndBurnable {
```

Comment:
Would probably name this either `RefImplIBCERC20` or `IBCERC20RefImpl` to make it clear that it's a reference implementation.

Reply:
done